index.js

```diff
@@ -45,10 +45,17 @@ module.exports = {
 
           const response = await self.apos.http.post(url, options);
           if (!response.success) {
+            self.logInfo('hcaptcha-invalid-token', {
+              ip: req.ip,
```

Comment:
Shouldn't we just pass `req` in this and similar cases, since we have it, rather than passing req.ip by hand? It's more futureproof.